sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -191,53 +228,57 @@ export class ConfirmationOfEnrollmentController {
    */
   @HasLocationAccess("locationId")
   @Patch(
-    ":locationId/confirmation-of-enrollment/application/:applicationId/confirm",
+    ":locationId/confirmation-of-enrollment/disbursement/:disbursementScheduleId/confirm",
   )
   async confirmEnrollment(
```

Comment:
Can we added some business explanation on how the COE confirmation works. Something that could help developer in the future to understand that this method will be called twice, for instance.

Reply:
Sure. I can add the comments accordingly.

Reply:
Comment added.

Reply:
Thanks for adding the comments, there is a type here "upto". Just a missing space 😉